client/src/redux/prop-types.ts

```diff
@@ -103,6 +103,7 @@ export type ChallengeWithCompletedNode = {
   isCompleted: boolean;
   order: number;
   superBlock: SuperBlocks;
+  tags: string[];
```

Comment:
We can declare these as a type union of strings (Ex: `type tags = “Header” | “Footer”`) but this would require hard-coding all of these tags with the assumption that we would reuse all of these across challenges). Or, would we have different type declarations based on the challenge?